modin/core/storage_formats/base/query_compiler.py

```diff
@@ -4588,6 +4588,18 @@ def rolling_sum(self, fold_axis, rolling_args, *args, **kwargs):
             self, rolling_args, *args, **kwargs
         )
 
+    @doc_utils.doc_window_method(
+        result="sem",
+        refer_to="sem",
+        params="""
+        *args : iterable
+        **kwargs : dict""",
```

Comment:
```suggestion
        **kwargs : dict,
```

I thought this would cause a syntax error 🤔